oxts_driver:
  ros__parameters:
#===============================================================================
    # INS configuration (set up to match the unit)
    ## Unit IP address
    unit_ip: 192.168.25.44
    ## Unit Port (3000 unless intentionally configured otherwise on the INS)
    unit_port: 3000

#===============================================================================
    # Configure ROS messages to be published by the node.
    ## Configure output rates of individual topics
    ## Rates must be factors of the ncom_rate, configured above.
    ## Rates are in Hz. 0 Hz => Message will not be published
    ### Configure sample rate of NCom (do not exceed output rate of ncom)
    ncom_rate               : 100.0